src/zepto.js

```diff
@@ -437,9 +437,10 @@ var Zepto = (function() {
 
     ready: function(callback){
       // don't use "interactive" on IE <= 10 (it can fired premature)
+      // pipe any ready callback call over task/event queue (#1235)
       if (document.readyState === "complete" ||
           (document.readyState !== "loading" && !document.documentElement.doScroll))
-        callback($)
+        setTimeout(function(){ callback($) })
```

Comment:
I wrote it in the such way because keep in mind that jQuery once deleted the 2nd argument, see bug [10417](https://bugs.jquery.com/ticket/10417). This situation is the same: we just want to execute it within event loop as soon as possible. Are there reasons to add 2nd argument? Code style?